<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telacadastrarModelo</class>
 <widget class="QDialog" name="telacadastrarModelo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditDescricao">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCadastrar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIncluirFoto">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>IncluirFoto</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFoto">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMarca">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVoltar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDescricao</tabstop>
  <tabstop>comboBoxMarca</tabstop>
  <tabstop>pushButtonIncluirFoto</tabstop>
  <tabstop>pushButtonCadastrar</tabstop>
  <tabstop>pushButtonVoltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
